kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/autoconfig/KafkaAutoConfiguration.java

```diff
@@ -160,6 +160,7 @@ public KafkaAutoConfiguration(KafkaProperties properties) {
     @Bean
     @ConditionalOnMissingBean
     @ConditionalOnBean({ConsumerFactory.class, KafkaMessageConverter.class, Fetcher.class})
+    @ConditionalOnProperty(value = "axon.kafka.consumer.event-processor-mode", havingValue = "TRACKING")
```

Comment:
We could, if the properties would allow defining a default `group.id` for the `SubscribableKafkaMessageSource`. 

As I have currently set it up that anybody is *required* to provide the `group.id`, I cannot auto configure it out of the box. We could/should brainstorm about this a little. Ideally we have some users of our upcoming RC2 which will come up with ideas how to auto configure a single `SubscribableKafkaMessageSource`. As an MVP-thought, I think it's fine we do not auto configure the `SubscribableKafkaMessageSource`.